broker-util/oo-stats

```diff
@@ -224,6 +229,18 @@ class OOStats
           end
           puts "\n\n"
         end
+      when :region
+        res[:region_summaries].sort_by {|a| a[:name] }.each do |region|
+          outline "Region '#{region[:name]}' summary:"
+          text_legendize region, @columns_for[:region_summary]
+          puts "\nNodes:"
+          text_tableize region[:nodes].sort_by {|a| a[:name]}, @columns_for[:node_table]
+          unless region[:missing_nodes].empty?
+            puts "\nWARNING: the following node(s) in this region DID NOT respond:"
+            puts region[:missing_nodes].join ", "
```

Comment:
missing nodes format:
{"nodexyz.rhcloud.com"=>{"name"=>"nodexyz.rhcloud.com"}}

This was my fix and it appears to work.
puts region[:missing_nodes].collect {|key,val| val["name"] }.join ", "